internal/winapi/zsyscall_windows.go

```diff
@@ -58,7 +61,10 @@ var (
 	procLogonUserW                             = modadvapi32.NewProc("LogonUserW")
 	procLocalAlloc                             = modkernel32.NewProc("LocalAlloc")
 	procLocalFree                              = modkernel32.NewProc("LocalFree")
-	procGetProcessImageFileNameW               = modkernel32.NewProc("GetProcessImageFileNameW")
```

Comment:
I assume this is just cleanup of unused code?

Reply:
Right. It was originally going to be used for getting stats for host process containers, but there's a call that returns basically everything we need all together including this